### PR TITLE
SUBMARINE-1201. Add buttons in workbench, fix bugs and beautify the pages

### PR DESCRIPTION
**Pull request by KUAN-HSUN-LI**
### What is this PR for?
1. Improve the workbench
2. Fix bugs
3. Add buttons
### What type of PR is it?
[Improvement]

### Todos
* [x] - Add register model button
* [x] - Add delete model version button
* [x] - Add create model serve and delete model serve buttons  
* [x] - Add delete registered model button
* [x] - Set the default language from 'zh_CN' to 'en_US'

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-1201

### Screenshots (if appropriate)
https://user-images.githubusercontent.com/38066413/155376183-10b33456-26f3-48cd-950c-594310fe947f.mp4

### Questions:
* Do the license files need updating? No
* Are there breaking changes for older versions? No
* Does this need new documentation? No

### CODE REVIEW

**Comment by KUAN-HSUN-LI** [comment]
@pingsutw @jeff-901 Help me review the PR. Thanks.

File: submarine-workbench/workbench-web/src/app/pages/workbench/model/model-home/model-form/model-form-tags/model-form-tags.component.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  }
+
+  handleInputConfirm = () => {
+    if (this.inputValue) {
+      const newTag = this.inputValue;
+      this.tags.push(newTag);
+    }
+    this.inputValue = "";
+    this.inputVisible = false;
+  }
```
Comment (jeff-901) [inline submarine-workbench/workbench-web/src/app/pages/workbench/model/model-home/model-form/model-form-tags/model-form-tags.component.ts:43]: Do you check if the user inputs a duplicate tag?

File: submarine-workbench/workbench-web/src/app/pages/workbench/model/model-home/model-form/model-form.component.ts [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    //     this.nzMessageService.success('Register Model Success!');
+    //     this.closeModal();
+    //   },
+    //   error: (msg) => {
+    //     console.log(msg)
+    //     this.nzMessageService.error(`Current model has been registered in this registered model.`, {
+    //       nzPauseOnHover: true,
+    //     });
+    //   },
+    // })
```
Comment (jeff-901) [inline submarine-workbench/workbench-web/src/app/pages/workbench/model/model-home/model-form/model-form.component.ts:119]: Make sure to delete the commented code.

Comment (KUAN-HSUN-LI) [reply submarine-workbench/workbench-web/src/app/pages/workbench/model/model-home/model-form/model-form.component.ts]: OK

**Review by pingsutw — APPROVED** [review]
LGTM, thanks

**Comment by pingsutw** [comment]
@KUAN-HSUN-LI Could you resolve the conflict? will +1 again, thanks

**Review by pingsutw — APPROVED** [review]

**Pull request closed without merging**